### PR TITLE
Add Rflink component documentation.

### PR DESCRIPTION
**Pull request by aequitas**
**Description:** Documentation for Rflink component/platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4547

### CODE REVIEW

**Comment by aequitas** [comment]
Updated to latest changes in component, ready for review.

**Comment by Genestealer** [comment]
Really looking forward to trying this out.

**Comment by aequitas** [comment]
@Genestealer You already can, just install the current development version :)

File: source/_components/light.rflink.markdown [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+*Dimmable*: Sends only `dim` and `off` commands. This does not work on on/off type devices as they don't understand the `dim` command. For dimmers this does not cause issues with signal repetitions.
+
+By default new lights are assigned the `switchable` type. Protocol supporting dimming are assigned the `hybrid` type. Currently only `newkaku` protocol is detected as dimmable. Please refer to Device Support to get your dimmers supported.
+
+# Hiding/ignoring lights
+Lights are added automatically when the Rflink gateway intercepts a 433Mhz command in the ether. To prevent cluttering the frontend use any of these methods:
+
+- Configure a `new_devices_group` for lights and optionally add it to a different `view`.
+- Hide unwanted devices using [customizations](https://home-assistant.io/getting-started/customizing-devices/)
```
Comment (Genestealer) [inline source/_components/light.rflink.markdown]: There is no need for the full link, just (/getting-started/customizing-devices) is enough.

File: source/_components/light.rflink.markdown [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- Ignore devices on a platform level: https://home-assistant.io/components/rflink/#ignoring-devices
```
Comment (Genestealer) [inline source/_components/light.rflink.markdown]: There is no need for the full link, just (/components/rflink/#ignoring-devices) is enough.

File: source/_components/sensor.rflink.markdown [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- **name** (*Optional*): Name for the device, defaults to Rflink ID.
+- **sensor_type_** (*Optional*): Override automatically detected type of sensor.
+- **unit_of_measurement** (*Optional*): Override automatically detected unit of sensor.
+- **aliasses** (*Optional*): Alternative Rflink ID's this device is known by.
+
+# Hiding/ignoring sensors
+Sensors are added automatically when the Rflink gateway intercepts a 433Mhz command in the ether. To prevent cluttering the frontend use any of these methods:
+
+- Configure a `new_devices_group` for sensors and optionally add it to a different `view`.
+- Hide unwanted devices using [customizations](https://home-assistant.io/getting-started/customizing-devices/)
```
Comment (Genestealer) [inline source/_components/sensor.rflink.markdown]: There is no need for the full link, just (/getting-started/customizing-devices) is enough.

File: source/_components/sensor.rflink.markdown [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- Ignore devices on a platform level: https://home-assistant.io/components/rflink/#ignoring-devices
```
Comment (Genestealer) [inline source/_components/sensor.rflink.markdown]: There is no need for the full link, just (/components/rflink/#ignoring-devices) is enough.

**Pull request merged**